### DISCUSSION TITLE
Tie two notes of different durations

### DISCUSSION THREAD

**Discussion by BlairBeverly** [CoderLine/alphaTab#1437] [Q&A] [answered]
Is it possible to tie two notes with different durations together (in piano tuning)? For example, these both throw errors for me:
Ab4.4 {ch "Fmi7"} Ab4.8 G4.8 G4.4{-} Bb4.8 Ab4.8
Ab4.4 {ch "Fmi7"} Ab4.8 G4.8 G4.4{t} Bb4.8 Ab4.8

**Comment by Danielku15** [comment, marked as answer]
The problem is likely similar to your request [here](https://github.com/CoderLine/alphaTab/discussions/1436). If you specify the tie on the single note (not on the beat effects) it should work. 

```
Ab4.4 {ch "Fmi7"} Ab4.8 G4.8 G4{-}.4 Bb4.8 Ab4.8 |
Ab4.4 {ch "Fmi7"} Ab4.8 G4.8 G4{t}.4 Bb4.8 Ab4.8
```

![image](https://github.com/CoderLine/alphaTab/assets/674916/188e7439-1d55-4690-94e6-75cb34cb927f)

I'm thinking it would be a nice idea to allow note effects in the place where beat effects are specified. The effect would then be applied to all notes in the beat.

**Reply by BlairBeverly** [reply]
Thank you so much- this worked! Personally, the current functionality completely meets my needs as is, but it might be helpful to include an example in the alphatex documentation that shows note effects and beat effects together.
